tensorflow/contrib/lite/examples/ios/simple/.gitignore

```diff
@@ -0,0 +1,63 @@
+# Xcode
```

Comment:
The examples in tensorflow/examples/ios had the gitignore one level higher. Maybe copy the one there?

https://github.com/tensorflow/tensorflow/blob/master/tensorflow/examples/ios/.gitignore

and be aware of https://github.com/tensorflow/tensorflow/pull/14638